pkg/build

```diff
@@ -12,22 +12,20 @@ WEBPACK_PACKAGES = \
 	metrics \
 	networkmanager \
 	packagekit \
+	pcp \
 	playground \
 	selinux \
 	shell \
 	sosreport \
+	ssh \
 	static \
 	storaged \
 	systemd \
 	tuned \
 	users \
 	$(NULL)
 
-MANIFESTS = \
-	$(WEBPACK_PACKAGES:%=$(srcdir)/dist/%/manifest.json) \
-	pkg/pcp/manifest.json \
-	pkg/ssh/manifest.json \
-	$(NULL)
```

Comment:
TBH like it better the current way -- pcp and ssh really aren't webpacks. They have manifests, yes, but that's not a webpack concept. It adds unnecessary build overhead, too.

Reply:
The problem with this is the `cockpit/` link goes to `dist/` which means that if we want to install these files with the `nobase_` rule, we need to figure out a way to get the manifests from `pkg/` to `dist/`.

Alternatives are:
 - copy.  This is what we used to do.
 - add two separate/simple rules (with corresponding `dir` variables) for installing these two files and completely remove them from the `MANIFESTS` list.

Reply:
Ah, I didn't realize that this commit was a prerequisite for the subsequent ones, or just standalone -- as by itself, the little clenaup isn't worth the overhead. If it helps to enable the compression-webpack plugin and such, it's ok for me.

Reply:
Dropped this after our chat: let's just add the 4 lines to the build system to deal with these manually.